src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -728,7 +729,21 @@ export class PatientViewPageStore {
     // set version 2 of the mutational signature as default
     @observable _selectedMutationalSignatureVersion: string =
         MutationalSignaturesVersion.V2;
+    // initialLoad to set _selectedMutationalSignatureVersion while loading the tab for the first time
+    @observable initialLoad: boolean = true;
```

Comment:
We can set `_selectedMutationalSignatureVersion` = undefined by default. and remove `initialLoad`, see more comments below.

Reply:
`_selectedMutationalSignatureVersion` has been set to undefined (and `initialLoad` removed) in PatientViewPageStore